projects/cm_mcu/CommandLineTask.c

```diff
@@ -808,7 +814,7 @@ static BaseType_t eeprom_write(int argc, char ** argv)
   data = strtoul(argv[2],NULL,16);
   addr = strtoul(argv[1],NULL,16);
   uint32_t block = EEPROMBlockFromAddr(addr);
-  if((block==1)|((EBUF_MINBLK<=block)&&(block<=EBUF_MAXBLK))){
```

Comment:
@gwengardner please look at this change too

Reply:
Oh right! Yes I agree!